inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/stream/InlongStreamFieldInfo.java

```diff
@@ -48,14 +48,16 @@
     @ApiModelProperty(value = "value expression of predefined field")
     private String preExpression;
 
+    @ApiModelProperty("Field type")
     private String fieldType;
 
+    @ApiModelProperty("Field comment")
     private String fieldComment;
 
+    @ApiModelProperty("Is source meta field, 0: no, 1: yes")
```

Comment:
update the comment "Is source meta field, 0: no, 1: yes" to "Is meta field, 0: no, 1:yes"